pandas/tests/reshape/test_pivot.py

```diff
@@ -38,18 +38,18 @@ def setup_method(self, method):
                                'E': np.random.randn(11),
                                'F': np.random.randn(11)})
 
-    def test_pivot_table(self):
+    def test_pivot_table(self, observed):
```

Comment:
can you add the example from the original issue (and assert the output frame).

Reply:
Added a check passing observed is equivalent to previous pivot_table call based on example data.